src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptionhandlers/ErrorNotificationExceptionHandler.java

```diff
@@ -33,24 +34,35 @@ public ErrorNotificationExceptionHandler(MessageAutoCompletor autoCompletor) {
         long deliveryCount,
         Throwable throwable
     ) {
-        if (throwable instanceof ErrorNotificationException) {
+        Throwable processedThrowable = processThrowable(throwable);
+
+        if (processedThrowable instanceof ErrorNotificationException) {
             return handleErrorNotificationException(
                 lockToken,
                 messageId,
                 deliveryCount,
-                (ErrorNotificationException) throwable
+                (ErrorNotificationException) processedThrowable
```

Comment:
may be worth logging a warning here as well.

Reply:
yes it is, in that method 😉